src/components/Square.js

```diff
@@ -1,19 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
+
+
 
 const Square = (props) => {
-  // For Wave 1 enable this 
-  //  Component to alert a parent 
-  //  component when it's clicked on.
+  
+  const squareClick = () => {
+    props.onClickCallback(props.id);
+  };
 
-  return <button
-    className="square"
-  >
-    {props.value}
-  </button>
-}
+  return <button onClick={squareClick} className='square' id={props.id}>{props.value}</button>;
```

Comment:
instead of creating a separate function you could do something like 
```suggestion
  return <button onClick={() => props.onClickCallback(props.id)} className='square' id={props.id}>{props.value}</button>;
```